openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -254,6 +254,7 @@ def post_placeholder_process(self, placeholder, failed):
 
     def delete_placeholder(self, placeholder):
         """Remove placeholder if building was successful"""
+        cmds.sets(placeholder.scene_identifier, rm= PLACEHOLDER_SET)
```

Comment:
```suggestion
        cmds.sets(placeholder.scene_identifier, rm=PLACEHOLDER_SET)
```

Also, **why** does this fix the issue? What was the original error?

Reply:
Because when you delete the only object in a set, the set is deleted. 
By removing the object and then deleting it, the set is kept.